AppKit/CPEvent.j

```diff
@@ -202,6 +226,28 @@ var _CPEventPeriodicEventPeriod         = 0,
 }
 
 /* @ignore */
+- (id)_initEnterExitEventWithType:(CPEventType)anEventType location:(CGPoint)aPoint modifierFlags:(unsigned)modifierFlags
```

Comment:
I agree with you but I've only reproduced what is already done for other types of events (mouse, key, other)...

I can change but it would then be more consistent to do so with all types of events.

What do you prefer ?